applications/GeoMechanicsApplication/tests/geo_beam_with_reset_displacement.gid/MeshStructure.svg

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
can both distances be places below the beam line?

Reply:
Done!